core/src/executor/evaluate/evaluated.rs

```diff
@@ -79,6 +93,9 @@ impl TryFrom<Evaluated<'_>> for HashMap<String, Value> {
             Evaluated::Value(Value::Str(v)) => HashMap::parse_json_object(v.as_str()),
             Evaluated::Value(Value::Map(v)) => Ok(v),
             Evaluated::Value(v) => Err(EvaluateError::MapOrStringValueRequired(v.into()).into()),
+            Evaluated::StrSlice { source, range } => {
```

Comment:
When writing StrSlice as a test, is it possible to try from that part?

Reply:
Added an example of using SUBSTR in the error.rs part of the schemaless test. 
Although it is for test coverage, I put a temporary test in anticipation of clear use in the future.